### DISCUSSION TITLE
remove or disable pre-rendered page without  run build

### DISCUSSION THREAD

**Discussion by MortezaNajafian** [vercel/next.js#22301] [Help] [answered]
Hello to all dear friends
I have a programming training site and I use SSG feature to increase speed and quality. But I have a problem, sometimes I need to temporarily disable some rendered pages. In NEXTJS, to do this, we must run the project again by Run Build and put the output on the host again.
How can I manage this and make the rendered page unavailable to users without building the project?
Thank You

**Comment by michrome** [comment, marked as answer]
If the page shouldn't always be available then, by definition, you _can't_ cache the page. As such, [server-side rendering](https://nextjs.org/docs/basic-features/typescript#static-generation-and-server-side-rendering) is probably a better fit because you can include the logic you need to determine whether or not the page should be available.

**Reply by MortezaNajafian** [reply]
thats right .thank you
